bpaotu/bpaotu/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+import logging
+logger = logging.getLogger("rainbow")
+
+
+class ImportLog(models.Model):
+    ontology_name = models.CharField(max_length=300)
+    import_result = models.TextField(null=True, blank=True)
```

Comment:
track the filename the error was in (and run `os.path.basename` on it)

Reply:
Fixed.